tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/passes/BoundedCheckerPassImpl.scala

```diff
@@ -129,7 +129,7 @@ class BoundedCheckerPassImpl @Inject() (
     val outcome = checker.run()
     rewriter.dispose()
     logger.info(s"The outcome is: " + outcome)
-    outcome == NoError()
+    outcome.isOk
```

Comment:
Isn't it easier to do this

```suggestion
    outcome == NoError || outcome == ExecutionsTooShort()
```

instead of adding an extra field to `CheckerResult`?

Reply:
Yeah, I also thought about this solution. But it would be good to have the condition where it is defined, that is, in the case classes of `CheckerResult`.

Reply:
I'd say it's the pass that decides what result is okay, but I won't block on this.
Maybe @Kukovec or @shonfeder have an opinion if they pass by.